components/CaseStatus/AddCaseStatusForm/AddCaseStatusForm.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import AddCaseStatusForm from './AddCaseStatusForm';
+import { render, fireEvent } from '@testing-library/react';
+import { residentFactory } from 'factories/residents';
+
+const mockedResident = residentFactory.build();
+
+describe('ReviewAddRelationshipForm - CIN', () => {
+  it('displays the form', () => {
+    const { getByText } = render(
+      <AddCaseStatusForm personId={mockedResident.id} prefilledField={{}} />
+    );
+
+    expect(getByText('Child in need')).toBeInTheDocument();
+  });
+
+  it('should disable the submit button when not completed', () => {
+    const { getByTestId } = render(
+      <AddCaseStatusForm personId={mockedResident.id} prefilledField={{}} />
+    );
+
+    expect(getByTestId('submit_button')).toHaveAttribute('disabled');
```

Comment:
```suggestion
    expect(getByTestId('submit_button')).toBeDisabled();
```

This provides a clearer expectation explanation when the test fails, see https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-the-wrong-assertion.